src/System.Private.CoreLib/shared/System/Security/SecureString.Unix.cs

```diff
@@ -49,14 +50,14 @@ private unsafe void InitializeSecureString(char* value, int length)
             byte* ptr = null;
             try
             {
-                _buffer.AcquirePointer(ref ptr);
+                _buffer!.AcquirePointer(ref ptr);
```

Comment:
If you add a Debug.Assert for `_buffer != null` before the try, you should avoid both `!` in these method.